client/components/Settings.js

```diff
@@ -15,20 +15,52 @@ import CustomBankField from './CustomBankField';
 import {OpCatChartTypeSelect, OpCatChartPeriodSelect} from './Charts';
 import T from './Translated';
 
-
 class Account extends React.Component {
 
+    constructor(props) {
+        super(props);
+        this.listener = this._listener.bind(this);
+    }
+
+    _listener() {
+        this.forceUpdate();
+    }
+
+    componentDidMount() {
+        store.on(State.settings, this.listener);
+    }
+
+    componentWillUnmount() {
+        store.removeListener(State.settings, this.listener);
+    }
+
     onDelete(id) {
         Actions.DeleteAccount(this.props.account);
     }
 
-    render() {
-        var a = this.props.account;
+    setAsDefault() {
+        Actions.SetDefaultAccount(this.props.account);
+    }
 
-        var label = a.iban ? `${a.title} (IBAN: ${a.iban})`
-                           : a.title;
+    render() {
+        let a = this.props.account;
+        let label = a.iban ? `${a.title} (IBAN: ${a.iban})` : a.title;
+        let defaultAccountTitle = t("settings.set_default_account") || "Set as default account";
+        let selected = "fa-star-o";
+
+        if (store.getDefaultAccountId() === this.props.account.id) {
+            defaultAccountTitle = "";
+            selected = "fa-star";
+        }
 
-        return <tr>
+        return <tr className="bankAccount">
```

Comment:
Well, it seems the class is actually never defined in the CSS. Maybe this can just be removed?